src/components/StudyNavBar/StudyNavBar.js

```diff
@@ -34,8 +34,8 @@ const StudyNavBar = ({match, history, isBeta}) => {
       endString: 'collaborators',
     },
     {
-      tab: 'Logs',
-      endString: 'logs',
+      tab: 'Events',
+      endString: 'events',
```

Comment:
If you're going to change this, you need to update the route, too. I think you're safe just keeping it as `events` as it only shows up in the url.

Reply:
i did in another commit